voc/application.go

```diff
@@ -25,9 +27,12 @@
 
 package voc
 
+// Application This encapsulates the whole (source) code of an application.
```

Comment:
maybe a "hack" to have slightly better auto-documented structs. Can we have something like `$entity is an entity in our Cloud ontology. $description_sentence`? Otherwise we could have some kind of convention that we should have `$Entity is...` descriptions in the ontology, but we will probably break it at some point.

Reply:
Very good idea.